.changeset/short-icons-wish.md

```diff
@@ -0,0 +1,8 @@
+---
+'fingerprint-pro-server-api-dotnet-sdk': patch
+---
+
+- Fix descriptions formatting:
```

Comment:
We don't need only schema-related updates

Reply:
Fixed